### ISSUE TITLE
PO042 false positive with UseEffectiveCount

### ISSUE THREAD

**Issue by guycall** [apigee/apigeelint#590]
Upgrading from v2.75.0 to v2.76.0 has introduced a new linting error in our project.

```
PO042 The element <UseEffectiveCount> is not allowed here in profile=apigee
```

This occurs in all our Spike Arrest policies. From a look at the docs our usage is correct and also these policies are deployed and appear to be working as expected. 

```
<SpikeArrest name="SA-ThrottleOnSomething">
  <Rate ref="somevariable">10ps</Rate>
  <UseEffectiveCount>true</UseEffectiveCount>
</SpikeArrest>
```
